Assets/Simon/Scripts/Colors.cs

```diff
@@ -0,0 +1,5 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+

```

Comment:
This file has only using directives? (no actual code)

Reply:
Done. I removed it.